src/components/avatar/index.tsx

```diff
@@ -187,6 +187,17 @@ const Avatar = forwardRef<any, AvatarProps>((props: AvatarProps, ref: React.Forw
     onPress,
     children
   } = themeProps;
+
+  const accessibleHitSlop = useMemo(() => {
+    const hitTargetPadding = Math.max(0, (48 - size) / 2);
```

Comment:
hitSlop can be a number by type.
react-native type: Rect or number, currently we are using Rect.
Let's use number instead since all the Rect values are identical.
Also when using number we don't need to memoize the object we can pass the Math.max to the hitSlop values itself